2022 Tip Updates (#9222)
tgstation/TerraGov-Marine-Corps: strings/tips/xeno.txt

Always thank your drones and hivelords for supporting the hive!
Do not fight as a young xenomorph.
Most alien deaths are caused by over-aggression. Rein yourself in, or you may find yourself dying round after round.
If you have difficulty clicking marines, try using bump slash. You can use movement keys to slash marines. That said, clicking marines to slash them is faster than bump slashing!
It is EXTREMELY IMPORTANT that you as a xenomorph have your abilities set up on keybinds. Ask for help if you don't know how in XOOC or mhelp.
In the xeno tab, you can use the crawl through vent verb to quickly find and use vents in the floor without having to alt-click them. You can also keybind it!

> It is faster to re-fill an empty acid well then rolling on the ground to put out a fire.

Reviewer: ```suggestion
It is faster to re-fill an empty acid well than rolling on the ground to put out a fire.
```

> Acid well extinguishes fire and explode into an acid cloud when destroyed. Queen, shrike, praetorian, boiler, and hivelord can make acid wells.

Reviewer: ```suggestion
Acid well extinguishes fire and explode into an acid cloud when destroyed. The queen, the shrike, praetorians, boilers, and hivelords can make acid wells.
```

> Queen, Shrike, and hivelord have access to Queen's Blessing, a menu for the hive to use psychic points to give xenomorphs a cutting edge in the battlefield.

Reviewer: ```suggestion
The queen, shrike, and hivelord have access to Queen's Blessing, a menu for the hive to use psychic points to give xenomorphs a cutting edge in the battlefield.
```

As a xeno, aging from young to mature gives you the most benefit, further aging has diminishing returns.
As a xeno, don't take too long attacking during the crash phase. The ship can be rigged to explode in twenty minutes, ending the round in a tie.
As a xeno, remember that neurotox injected by defilers, sentinels, and the Queen can overdose, killing a marine quickly. Try to avoid this when capturing, if they scream in pain they've probably had enough.
As a xeno, you can crawl through holes in walls made by acid by SpecialClicking (Default: CTRL) on them or click-dragging yourself on them. If you're too big to pass through, you'll damage the wall instead.
As a xeno, you can climb over window frames by SpecialClicking (Default: CTRL) on them or click-dragging yourself on them.
As a xeno, do not camp the landing zones prior to the marines building their defenses.
As a xeno, do not block other xenos' paths when they are retreating. Circumvent this by being on Help Intent or by simply moving away.
As a xeno, white weed helps you heal faster, green weeds slows down marines, and weed with no colors are normal weeds. Rest on the white weeds to heal faster!

> As a larva, ask your hive leader what you should evolve into.

Reviewer: ```suggestion
As a larva, ask your hive leader what you should evolve into, if nobody responds you can check hive status and think what xenomorphs need the most.
```
probably needs better wording but it's niight.

As a larva, you can slide under unbolted and unwelded doors by clicking on the door.
As a larva, stay on weeds while growing, you will grow faster as a result.
As a runner or larva, you can pass over window frames without climbing on to them beforehand.
As a drone or sentinel, your corrosive acid is very weak, but it's better than nothing.

> As a drone, spread weeds as much as you can, especially near the front line fighters.

Reviewer: ```suggestion
As a drone or hivelord, spread weeds as much as you can, especially near the front line fighters.
```

Always thank your drones and hivelords for supporting the hive!

> Do not fight as a young xenomorphs.

Reviewer 2: ```suggestion
Do not fight as a young xenomorph.
```

> Most alien deaths are caused by over-aggression. Rein yourself in, or you may find yourself dying round after round.

Author: "_Rein in_ is an allusion /  figure of speech to pulling on the reins of a horse in order to exercise more control."

If you have difficulty clicking marines, try using bump slash. You can use movement keys to slash marines. That said, clicking marines to slash them is faster than bump slashing!

> A good hive consists of having a hivelord and a gorger to heal xenomorphs' wounds and give xenomorphs some pregame before engaging marines.

Reviewer 2: :/

As a xeno, don't take too long attacking during the crash phase. The ship can be rigged to explode in twenty minutes, ending the round in a tie.
As a xeno, remember that neurotox injected by defilers, sentinels, and the Queen can overdose, killing a marine quickly. Try to avoid this when capturing, if they scream in pain they've probably had enough.
As a xeno, you can use the crawl through vent verb found in the xeno tab to quickly find and use vents in the floor without having to alt-click them.
As a xeno, you can crawl through holes in walls made by acid by SpecialClicking (Default: CTRL) on them or click-dragging yourself on them. If you're too big to pass through, you'll damage the wall instead.
As a xeno, you can climb over window frames by SpecialClicking (Default: CTRL) on them or click-dragging yourself on them.
As a xeno, do not camp the landing zones prior to the marines building their defenses.
As a xeno, take advantage of acid wells, they put out fire and explode into an acid cloud when destroyed.
As a xeno, do not block other xenos' paths when they are retreating. Circumvent this by being on Help Intent or by simply moving away.

> As a xeno, it is faster to re-fill an empty acid well then rolling on the ground to put out a fire.

Reviewer 2: ```suggestion
As a xeno, it is faster to re-fill an empty acid well than rolling on the ground to put out a fire.
```

> As a xeno, if you have access to Queen's Blessing, check how much psychic points you have so that you can purchase stuff to beat the marines.

Reviewer 2: phrasing

> As a xeno, if you have acid spit, you can spit at or shoot a welding fuel tank to blow it up next to hostiles.

Reviewer 2: ? phrasing

As a xeno, white weed helps you heal faster, green weeds slows down marines, and weed with no colors are normal weeds. Rest on the white weeds to heal faster!
As a larva, ask your hive leader what you should evolve into.
As a larva, you can slide under unbolted and unwelded doors by clicking on the door.
As a larva, stay on weeds while growing, you will grow faster as a result.
As a runner or larva, you can pass through window frames without climbing on to them beforehand.
As a drone or sentinel, your corrosive acid is very weak, but it's better than nothing.

> As a drone, spread weeds as much as you can, especially to the front line fighters.

Reviewer 2: ```suggestion
As a drone, spread weeds as much as you can, especially near the front line fighters.
```

Before you evolve as the Hivemind as a drone, hide into a secure area! Once you evolve as a hivemind, you cannot move anymore on the spot.

> As a carrier, you can hit a host directly with a thrown facehugger to have a high chance of infecting them near instantly. This will take time, however.

Reviewer: carrier throws facehugger > picks up said facehugger > carrier finds knocked down marine > carrier clicks on marine and plants a facehugger directly at the person (which takes time)

now i am not sure if it is the case with other facehuggers, not just larval

As a carrier, you can hide traps beneath certain objects and items.
As a hunter, you have almost perfect stealth if you stand still.
As a hunter, you can sneak / change move intention during stealth to make it hard on marines to see you while moving in stealth.

> As a ravager, targeting arms or legs is better to inflict a lot of damage while charging.

Reviewer 2: phrasing

As a ravager or the Queen, you are fire-resistant, NOT fireproof. You can still get lit on fire, but you take very little damage from it.
As a spitter, boiler, praetorian, or the Queen take advantage of unguarded barricades and defenses, melt or constantly spit acid on them if you can.
As a boiler or the Queen, you have the most powerful corrosive acid for melting things.

> As a boiler, shots have a small chance to bypass barricades, aim where you want them to land in case this happens.

Reviewer 2: phrasing
